pkg/storaged/format-dialog.jsx

```diff
@@ -330,9 +373,13 @@
                 let new_crypto_options;
                 if (is_encrypted(vals)) {
                     let opts = [];
-                    if (vals.mount_options &&
-                        (!vals.mount_options.auto || vals.mount_options.never_auto)) {
-                        opts.push("noauto");
+                    if (is_filesystem(vals)) {
+                        if (!mount_now || vals.at_boot == "never")
+                            opts.push("noauto");
+                        if (vals.at_boot == "nofail")
+                            opts.push("nofail");
+                        if (vals.at_boot == "netdev")
+                            opts.push("_netdev");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17888-20221124-142402-f4c856ed-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)